balancer/frontend-v2
# spike: Use API pricing

# Description

- Fetch prices from the new API instead of using Coingecko. 
- When a token is missing a price, log an error to Sentry so we can investigate and add it to the list if warranted. 

This requires https://github.com/balancer/tokenlists/pull/38 and/or https://github.com/beethovenxfi/beethovenx-backend/pull/301 to be merged before it will work completely. 

## Type of change


## How should this be tested?

- Ensure prices are being fetched from the API correctly, there should be a console log of `Fetched 591 prices`. 
- Ensure prices on individual pool pages are showing correctly in the tokens breakdown and trades list. 

## Checklist:



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204111333507466
